html/rendering/non-replaced-elements/the-fieldset-element-0/fieldset-default-style.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<title>fieldset default style</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<style>
+#ref {
+  display: block;
+  margin-left: 2px;
+  margin-right: 2px;
+  /* TODO replace above declarations with these when they are widely supported.
+  margin-inline-start: 2px;
+  margin-inline-end: 2px;
+  */
+  border: groove 2px ThreeDFace;
+  padding: 0.35em 0.75em 0.625em 0.75em;
+  /* TODO replace above declarations with these when they are widely supported.
+  padding-block-start: 0.35em;
+  padding-inline-end: 0.75em;
+  padding-block-end: 0.625em;
+  padding-inline-start: 0.75em;
+  */
+  min-width: min-content;
+  /* TODO change the above to min-inline-size per https://github.com/whatwg/html/pull/3914 */
```

Comment:
Should this be changed already? I think Fx is close to shipping this at least.

Reply:
min-inline-size is not yet supported in Safari or Edge, but I can update the comment.